content/id/docs/concepts/storage/persistent-volumes.md

```diff
@@ -0,0 +1,702 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- thockin
+- msau42
+title: Persistent Volume
+feature:
+  title: Storage orchestration
+  description: >
+    Automatically mount the storage system of your choice, whether from local storage, a public cloud provider such as <a href="https://cloud.google.com/storage/">GCP</a> or <a href="https://aws.amazon.com/products/storage/">AWS</a>, or a network storage system such as NFS, iSCSI, Gluster, Ceph, Cinder, or Flocker.
```

Comment:
Untranslated?

Reply:
Left over english, please translate this

Reply:
Sorry for that! Resolved in https://github.com/kubernetes/website/pull/14532/commits/d83e05dbfdbf54847b37ef7db85d8b9bd3bfaffe